CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# 0.9.0
+
+## New Features
+
+* General restructure of the classes thanks to the
+[Red Hat Open Source Contest](https://research.redhat.com/red-hat-open-source-contest/)
+project done by @mfocko.
+    * Classes are better linked together.
+    * Functionality is moved to the classes from the `GitProject` classes.
+    * Properties have also setters so you can now edit objects through them.
```

Comment:
"through them" seems like czechism to me. Do you want to say that you can now use the properties (setters) to modify objects?

Reply:
Changed.